code/modules/mob/living/simple_animal/abnormality/waw/silence.dm

```diff
@@ -40,21 +46,23 @@
 	. = ..()
 	if(meltdown_cooldown < world.time)
 		meltdown_cooldown = world.time + meltdown_cooldown_time
-		sound_to_playing_players('sound/abnormalities/silence/bong.ogg')	//Church bells ringing, whether it happens or not.
```

Comment:
Part of this WAS to alert people that it's time to work on Price of Silence again without having to be glued to a timer on another screen. I'm not saying this has to be kept in but it would be nice if we had some way to track it in game.

Reply:
I'll put the ambience sound here instead.